x-pack/agent/_meta/agent.docker.yml

```diff
@@ -8,12 +8,21 @@ outputs:
     username: elastic
     password: changeme
 
-streams:
-  - type: event/file
-    enabled: true
-    paths:
-      - /var/log/hello1.log
-      - /var/log/hello2.log
+datasources:
+  - namespace: sample
+    use_output: default
+    inputs:
+      - type: logs
+        processors:
+        streams:
+          - dataset: sample.acccess
+            paths: /var/log/sample/access.log
+          - enabled: false
+            dataset: sample.error
+            paths: /var/log/sample/error.log
+      - type: metrics/system
```

Comment:
it should be `system/metrics` as defined in https://github.com/elastic/beats/blob/feature-ingest/x-pack/agent/docs/agent_configuration_example.yml#L56